extensions.tex

```diff
@@ -79,12 +79,44 @@ \subsection{Beta Distribution Bag-of-words}
   \subsection{Grid Searching}
   \label{sec:grid_search}
 
-    \subsubsection{RNN}
-    \label{sec:rnn_grid_search}
+  \subsubsection{RNN}
+  \label{sec:rnn_grid_search}
 
-\subsection{Ensemble}
-\label{sec:ensemble}
+  \subsection{Ensemble}
+  \label{sec:ensemble}
+  
+  \subsection{Translation}
+  \label{sec:translation}
+  To try and improve the results from some of the classifiers the data an enhanced data set was created. To create the enhanced data set the original sentences were translated to and from an alternative language. This is from suggestions from a differing Kaggle competition 'Toxic Comments' \cite{Jigsaw2017}. 
```

Comment:
I'm afraid there are a few grammatical errors in this paragraph.
I believe what you're trying to say is:
"To improve the results of some classifiers, the dataset was enhanced by adding translations of the sentences through an intermediate language back to English. This method was shown to work in a previous Kaggle competition, 'Toxic Comments'"

You should also mention George's comment on how these are actually different applications. (This then means you can lead into the next paragraph on why we figured it was a good idea to try it still)